src/components/navigation/nav-tab-panel.sass

```diff
@@ -26,8 +26,8 @@
     background-color: $charcoal-light-6
     border-radius: 0 6px 0 0
     border-style: solid
-    border-color: $charcoal-light-1
-    border-width: 1.5px 1.5px 1.5px 0
+    border-color: $charcoal-light-2
+    border-width: 2px 2px 0 0
```

Comment:
Did these borders change to 2px in the spec?  The 1.5px seemed to cause weird render issues.  Glad to see this change.

Reply:
I took it from https://app.zeplin.io/project/5d62a554d64a9e02dcad80de/screen/5f7b52e62e06e6696a735f06 It looked like the badge had the same border thickness as the tabs
![image](https://user-images.githubusercontent.com/7716653/95368787-7157fc00-088b-11eb-95fe-6516f561f51d.png)